mdpow/restart.py

```diff
@@ -21,12 +21,9 @@
 
 .. autofunction:: checkpoint
 """
-from __future__ import absolute_import, print_function
-
-from six.moves import cPickle as pickle
+import _pickle as pickle
```

Comment:
Why `_pickle` — isn't there just `import pickle` in 3??

Reply:
Oh yeah, didn't realize I could just get pickle, I think _pickle is what showed up when I looked up cPickle replacement in python 3.